crates/oxc_parser/src/js/module.rs

```diff
@@ -265,6 +265,10 @@ impl<'a> Parser<'a> {
 
     // export Declaration
     fn parse_export_named_declaration(&mut self) -> Result<Box<'a, ExportNamedDeclaration<'a>>> {
+        while self.cur_kind() == Kind::At {
```

Comment:
it seems `eat_decorators` already has a while loop inside it?

Reply:
Otherwise you'll only save one decorator.

Reply:
Yes, `eat_decorator` already has a while loop inside it.

Reply:
Added reference and use `if` instead of `while` in [5f31f04](https://github.com/Boshen/oxc/pull/37/commits/5f31f04ae8f9b9128a22995ef777c767dc8234b1)

Reply:
https://github.com/Boshen/oxc/blob/4c6407b152257746d685ac553eabcd4898f11b9c/crates/oxc_parser/src/ts/statement.rs#L487-L489

Looks like it also checks for `@`